project/frontend/src/containers/Competition/Competition.js

```diff
@@ -3,120 +3,121 @@ import classes from "./Competition.module.css";
 import Block from "../../components/UI/Block/Block";
 import RankingsList from "../../components/RankingsList/RankingsList";
 import ChartCard from "../../components/ChartCard/ChartCard";
-import Rank from "../../components/Rank/Rank"
+import Rank from "../../components/Rank/Rank";
 
 const Competition = (props) => {
-    const toDayHourMinute = (totalTime) => {
-        let millisInDay = 24 * 60 * 60 * 1000,
-          millisInHour = 60 * 60 * 1000,
-          days = Math.floor(totalTime / millisInDay),
-          hours = Math.floor((totalTime - days * millisInDay) / millisInHour),
-          minutes = Math.round(
-            (totalTime - days * millisInDay - hours * millisInHour) / 60000
-          ),
-          pad = function (n) {
-            return n < 10 ? "0" + n : n;
-          };
-        if (minutes === 60) {
-          hours++;
-          minutes = 0;
-        }
-        if (hours === 24) {
-          days++;
-          hours = 0;
-        }
-        return [days, pad(hours), pad(minutes)];
+  const toDayHourMinute = (totalTime) => {
+    let millisInDay = 24 * 60 * 60 * 1000,
+      millisInHour = 60 * 60 * 1000,
+      days = Math.floor(totalTime / millisInDay),
+      hours = Math.floor((totalTime - days * millisInDay) / millisInHour),
+      minutes = Math.round(
+        (totalTime - days * millisInDay - hours * millisInHour) / 60000
+      ),
+      pad = function (n) {
+        return n < 10 ? "0" + n : n;
```

Comment:
what in god's name does this do

Reply:
soo if it's 11:05 (as in 11 hours and 5 minutes) it pads 5 minutes to 05 minutes

Reply:
wait isn't this your code? XD I didn't touch that file at all, it's just my formatter changing the formatting no code changes